components/brave_shields/browser/adblock_stub_response.h

```diff
@@ -17,7 +17,8 @@ namespace brave_shields {
 
 // Intercepts certain requests and blocks them by silently returning 200 OK
 // and not allowing them to hit the network.
-void MakeStubResponse(const network::ResourceRequest& request,
+void MakeStubResponse(const std::string& redirect,
```

Comment:
Can we rename it into something like `data_url` and use `base::Optional` to underline that it is not always provided?